### PR TITLE
Stellar: add support for StellarManageBuyOfferOp and StellarPathPaymentStrictSendOp.

### PR DESCRIPTION
**Pull request by overcat**
continuation of https://github.com/trezor/trezor-firmware/pull/1838
- updated trezor-common
- rename StellarManageOfferOp to StellarManageSellOfferOp
- rename StellarPathPaymentOp to StellarPathPaymentStrictReceiveOp
- rename StellarCreatePassiveOfferOp to StellarCreatePassiveSellOfferOp
- add support for StellarManageBuyOfferOp
- add support for StellarPathPaymentStrictSendOp

In c933d23079047ec9b97e3213a0265c6e5870d777, I only submitted changes related to Stellar.

I hope I didn't miss anything else, please let me know if there is any.

### CODE REVIEW

File: src/js/core/methods/helpers/stellarSignTx.js [hunk 1]
```diff
@@ -87,10 +87,10 @@ const transformOperation = (op: StellarOperation): ?StellarOperationMessage => {
                 amount: op.amount,
             };
 
-        case 'pathPayment':
+        case 'pathPaymentStrictReceive':
             validateParams(op, [{ name: 'destAmount', type: 'amount', obligatory: true }]);
```
Comment (overcat) [inline src/js/core/methods/helpers/stellarSignTx.js:91]: I am curious why only `destAmount` is checked here, but `sendMax` is not?

**Review by mroz22 — APPROVED** [review]
LGTM.
Tests are passing (except for cardano but this is expected, there were some more changes in firmware that will be reflected in another PR). 
I shall merge it tomorrow I think.

File: src/js/core/methods/helpers/stellarSignTx.js (hunk 1 above)
Comment (mroz22) [reply src/js/core/methods/helpers/stellarSignTx.js:91]: probably somebody forgot to add this? I guess this is not a big deal, firware will run its own validations.

**Comment by overcat** [comment]
Rebased.

**Pull request merged**